.github/workflows/conformance-multi-pool.yaml

```
@@ -0,0 +1,195 @@
+name: ConformanceMultiPoolIPAM
+
+# Any change in triggers needs to be reflected in the concurrency group.
+on:
+  pull_request:
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+  push:
+    branches:
+      - main
+      - ft/main/**
+    paths-ignore:
+      - 'Documentation/**'
+      - 'test/**'
+
+permissions: read-all
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.event.after }}
+  cancel-in-progress: true
+
+env:
+  # renovate: datasource=github-releases depName=cilium/cilium-cli
+  cilium_cli_version: v0.14.7
+  cilium_cli_ci_version:
+  kind_version: v0.17.0
```

Any reason not to use the latest version? 
FWIW, renovate will take care of updating these in the future cf. https://github.com/cilium/cilium/pull/26312, so would be nice to add the annotation as well:

```suggestion
  # renovate: datasource=github-releases depName=kubernetes-sigs/kind
  kind_version: v0.20.0
```

---

I wanted to use the same kind version as the other workflows currently in use. I have not yet tested newer versions in CI, but with `kind.sh` I had issues with newer versions of kind that I have not yet triaged (iirc they were dual-stack related).

In general I consider the kind upgrade to change unrelated to this PR.

---

Having said that, I'll make sure to add `# renovate: datasource=github-releases depName=kubernetes-sigs/kind` and check if v0.19 works

---

> In general I consider the kind upgrade to change unrelated to this PR.

Fully agree. Let's leave it at v0.17.0 and add the renovate annotation and then let's figure out the upgrade on the respective bot PR.